antenna/util.py

```diff
@@ -190,24 +191,16 @@ def sanitize_dump_name(val):
 
 
 class MaxAttemptsError(Exception):
-    """Maximum attempts error.
+    """Maximum attempts error."""
 
-    .. :py:attribute:: msg
-       message for the exception
-
-    .. :py:attribute:: return_value
-       The last return value for the function.
-
-    """
-
-    def __init__(self, msg, ret):
+    def __init__(self, msg, ret=None):
         super().__init__(msg)
         self.return_value = ret
 
 
 def wait_time_generator():
     """Return generator for wait times."""
-    for amt in [1, 1, 5, 10, 30]:
+    for amt in [1, 1, 5, 10, 10]:
```

Comment:
Changed the last item from 30 to 10 because waiting 30 seconds for anything seems unreasonable as a default.

I'm wondering whether a better default would be something like `[1, 1, 1, 1, 1]` or `[2, 2, 2, 2, 2]`.